pagerduty/pagerduty.go

```diff
@@ -45,33 +45,34 @@ type Logger struct {
 
 // Client manages the communication with the PagerDuty API
 type Client struct {
-	baseURL                    *url.URL
-	client                     *http.Client
-	FileLogger                 *Logger
-	Config                     *Config
-	Abilities                  *AbilityService
-	Addons                     *AddonService
-	EscalationPolicies         *EscalationPolicyService
-	Extensions                 *ExtensionService
-	MaintenanceWindows         *MaintenanceWindowService
-	Rulesets                   *RulesetService
-	EventOrchestrations        *EventOrchestrationService
-	EventOrchestrationPaths    *EventOrchestrationPathService
-	Schedules                  *ScheduleService
-	Services                   *ServicesService
-	Teams                      *TeamService
-	ExtensionSchemas           *ExtensionSchemaService
-	Users                      *UserService
-	Vendors                    *VendorService
-	EventRules                 *EventRuleService
-	BusinessServices           *BusinessServiceService
-	ServiceDependencies        *ServiceDependencyService
-	Priorities                 *PriorityService
-	ResponsePlays              *ResponsePlayService
-	SlackConnections           *SlackConnectionService
-	Tags                       *TagService
-	WebhookSubscriptions       *WebhookSubscriptionService
-	BusinessServiceSubscribers *BusinessServiceSubscriberService
+	baseURL                         *url.URL
+	client                          *http.Client
+	FileLogger                      *Logger
+	Config                          *Config
+	Abilities                       *AbilityService
+	Addons                          *AddonService
+	EscalationPolicies              *EscalationPolicyService
+	Extensions                      *ExtensionService
+	MaintenanceWindows              *MaintenanceWindowService
+	Rulesets                        *RulesetService
+	EventOrchestrations             *EventOrchestrationService
+	EventOrchestrationPaths         *EventOrchestrationPathService
+	ServiceEventOrchestrationActive *ServiceEventOrchestrationActiveService
```

Comment:
I'd maybe name this `ServiceEventOrchestrationStatus` to avoid having a field that is an adjective while all others are nouns